envergo/evaluations/migrations/0038_alter_request_submitted.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.2.13 on 2024-06-27 09:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("evaluations", "0037_request_submitted"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="request",
+            name="submitted",
+            field=models.BooleanField(default=False, verbose_name="Submitted"),
+        ),
+    ]
```

Comment:
You could add a single migration instead of 2

Reply:
En fait, je voulais que la valeur pour les demandes existantes soient bien « True » mais que le défaut pour les nouvelles évals soit « False ». Je ne sais pas si c'est possible avec une seule migration.

Reply:
Ah OK, je comprends.
J'imagine que tu aurais pu ajouter du code python pour le faire dans la même migration, mais ca aurait été du travail pour pas grand chose :
```
    operations = [
        migrations.AlterField(
            model_name="request",
            name="submitted",
            field=models.BooleanField(default=False, verbose_name="Submitted"),
        ),
        migrations.RunPython(set_existing_to_submitted, migrations.RunPython.noop),
    ]
```

Reply:
Ma paresse naturelle l'a emporté :)